api/src/main/java/org/pmiops/workbench/api/OfflineRdrExportController.java

```diff
@@ -127,8 +134,10 @@ private void createAndPushTask(List<Long> ids, String queuePath, String taskUri)
     } catch (IOException ex) {
```

Comment:
drive-by nit: Above, I'd declare a `TaskBuilder` and then use `taskBuilder.build()` in the cal to `createTask()`. Or just inline the builder.

Reply:
Done